public/index.ejs

```diff
@@ -267,10 +283,19 @@ function updateFollowingUI(users) {
       e.preventDefault();
       history.pushState({path: newURL.toString()}, '', newURL.toString());
       updateUIToMatchURL();
+      fetchAbortHandler();
       update();
     }
   }
 }
+
+function fetchAbortHandler() {
+  // abort all fetches in list
+  repoURLAbortControllersList.forEach((item) => {
```

Comment:
This list will keep growing forever right? We should probably pop from it.

And secondly, I guess this means we'll also be aborting each AbortController even for the fetches that have already completed? I think that's a no-op so it's probably fine for this hacky little tool, but let's add some documentation saying that.

Reply:
Ah yeah, 🤦, I can pop from the array after aborting.

Reply:
Certainly could. We could do it in a follow-up too.